### PR TITLE
[CHORE] Testnet config

### PR DESCRIPTION
**Pull request by mmrozek**
# Description

Initial config for the private testnet:

1. Node syncs using regular sync
2. Mining is enabled
3. Phoenix fork config will be a base for chain
4. Disabled features:
- pruning
- discovery
- blacklisting

### CODE REVIEW

**Review by kreisys — APPROVED** [review]
I cherry-picked the commit on top of `chore/update-sbt-add-nix` and it seems to make my problems go away so :shipit: plz

File: src/universal/conf/private-testnet.conf [hunk 1]
```diff
@@ -12,12 +12,21 @@ mantis {
...
+        mode = "archive"
+    }
   }
 
   blockchains {
     network = "private"
   }
 
+  consensus {
+    coinbase = "0011223344556677889900112233445566778899" # has to be changed for each node
```
Comment (mirkoAlic) [inline src/universal/conf/private-testnet.conf:26]: why this is part of the default?

File: src/main/resources/chains/private.conf [hunk 2]
```diff
@@ -1,8 +1,20 @@
 {
-  include "base.conf"
-  chain-id = "0x2A"
-  network-id = 42
-  custom-genesis-file = "chains/private-genesis.json"
-  dao = null
-  bootstrap-nodes = []
+    include "base.conf"
+    frontier-block-number = "0"
```
Comment (KonradStaniec) [inline src/main/resources/chains/private.conf:3]: I would double check if we do not have some boolean conflicts when are block numbers are equal to 0 (like in ets test sometimes we have fork block equal to `-1` to keep the ordering between forks)

**Review by KonradStaniec — APPROVED** [review]
Small comment to double check, also you need to sign your commit :)

File: src/universal/conf/private-testnet.conf (hunk 1 above)
Comment (mmrozek) [reply src/universal/conf/private-testnet.conf:26]: I wanted to have anything here(it is the default from application.conf). For each node, we have to obtain a different account

**Pull request merged**